elastic/kibana
# [EPM] Store map visualizations from the package registry and use saved object ID

This PR adds maps to the types of kibana assets that can be created from a package.

## Example map working
![image](https://user-images.githubusercontent.com/56361221/78067786-4da87c00-7365-11ea-8831-2b80c5d6903f.png)

## Testing
To test this change:

1. Open the Ingest Manager plugin
2. Install the Okta package

Kibana may crash because of the ingest pipeline within the okta package (this is happing on master without my change as well), this is what was happening to me
```
{ Error: Internal Server Error
    at respond (/Users/jbuttner/proj/es/kibana/node_modules/elasticsearch/src/lib/transport.js:349:15)
    at checkRespForFailure (/Users/jbuttner/proj/es/kibana/node_modules/elasticsearch/src/lib/transport.js:306:7)
    at HttpConnector.<anonymous> (/Users/jbuttner/proj/es/kibana/node_modules/elasticsearch/src/lib/connectors/http.js:173:7)
    at IncomingMessage.wrapper (/Users/jbuttner/proj/es/kibana/node_modules/elasticsearch/node_modules/lodash/lodash.js:4929:19)
    at IncomingMessage.emit (events.js:203:15)
    at endReadableNT (_stream_readable.js:1145:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  status: 500,
  displayName: 'InternalServerError',
  message: 'Internal Server Error',
  path: '/_ingest/pipeline/logs-okta.system-0.0.1',
  query: undefined,
  body:
   '---\nerror:\n  root_cause:\n  - type: "not_x_content_exception"\n    reason: "Compressor detection can only be called on some xcontent bytes or compressed\\\n      \\ xcontent bytes"\n  type: "not_x_content_exception"\n  reason: "Compressor detection can only be called on some xcontent bytes or compressed\\\n    \\ xcontent bytes"\nstatus: 500\n',
  statusCode: 500,
  response:
   '---\nerror:\n  root_cause:\n  - type: "not_x_content_exception"\n    reason: "Compressor detection can only be called on some xcontent bytes or compressed\\\n      \\ xcontent bytes"\n  type: "not_x_content_exception"\n  reason: "Compressor detection can only be called on some xcontent bytes or compressed\\\n    \\ xcontent bytes"\nstatus: 500\n',
  toString: [Function],
  toJSON: [Function] }

Terminating process...
 server crashed  with status code 1
```
The dashboard and map should still be installed even though kibana crashed.
3. Restart Kibana but not ES
4. Go to the dashboard page
5. Open the `[Filebeat Okta] Overview` dashboard
6. See that the map is now in the dashboard

## With the change
![image](https://user-images.githubusercontent.com/56361221/78166300-8a827a80-741a-11ea-84b4-afaad7bbf091.png)

## Without this change
![image](https://user-images.githubusercontent.com/56361221/78167441-7d668b00-741c-11ea-8d8b-c57a83400138.png)
